packages/kbn-ftr-common-functional-ui-services/kibana.jsonc

```diff
@@ -1,5 +1,5 @@
 {
-  "type": "shared-common",
+  "type": "shared-server",
```

Comment:
What makes this a "shared-server"?

Reply:
It imports `@kbn/ftr-common-functional-services`, which is server-only

Reply:
Thinking further about this... maybe they are type `test-helper`? Although they resolve to the `common` scope...

cc @elastic/kibana-operations, any thoughts?